Library/Homebrew/cmd/style.rb

```diff
@@ -120,7 +120,7 @@ def check_style_impl(files, output_type, options = {})
       # exitstatus can also be nil if RuboCop process crashes, e.g. due to
       # native extension problems.
       # JSON needs to be at least 2 characters.
-      if !status.success? || json.to_s.length < 2
+      if status.exitstatus > 1|| json.to_s.length < 2
```

Comment:
Rubocop exits with a 1 status code if offenses are found and a 0 exit code if there are no offenses. A convenient way of handling this in compact code might be to use:
```
if !status.exitstatus === 1..0 || json.to_s.length < 2
```